public/app/features/datasources/components/EditDatasSourceSubtitle.tsx

```diff
@@ -0,0 +1,81 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Badge, InlineField, InlineSwitch, useStyles2 } from '@grafana/ui';
+
+interface Props {
+  subTitle?: string;
+  isDefault: boolean;
+  alertingSupported: boolean;
+  onDefaultChange: (value: boolean) => void;
+}
+
+export function EditDataSourceSubtitle({ subTitle, isDefault, alertingSupported, onDefaultChange }: Props) {
+  const styles = useStyles2(getStyles);
+  const dataSourceType = subTitle ? getDataSourceType(subTitle) : '';
+  return (
+    <div className={styles.subTitleRow}>
+      <div className={styles.subTitleRowGroupFirst}>
+        <div className={styles.subTitle}>
+          Type: <span>{dataSourceType}</span>
+        </div>
+        <InlineField
+          label="Default"
+          tooltip="This datasource is used when you select the data source in panels. The default data source is
+        'preselected in new panels."
+          transparent={true}
+          labelWidth={8}
+          disabled={false}
+          className={styles.defaultDataSourceSwitch}
+        >
+          <InlineSwitch
+            id="basic-settings-default"
+            transparent={true}
+            value={isDefault}
+            onChange={(event: React.FormEvent<HTMLInputElement>) => onDefaultChange(event.currentTarget.checked)}
```

Comment:
Getting a warning here due to uncontrolled input, not sure why really. isDefault has a value here 
![image](https://user-images.githubusercontent.com/10999/234787602-24811ec9-5e9f-4fc0-85f6-1b31bba41e4d.png)